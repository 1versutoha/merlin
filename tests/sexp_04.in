["find", "use", "sexplib"]
["tell", "source", "module Test_sexp = struct "]
["tell", "source", "exception E of int with sexp "]
["tell", "source", "end"]
["tell", "end"]
["errors"]
["dump", "env"]
