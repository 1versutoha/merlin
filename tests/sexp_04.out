["return",true]
["return",null]
["return",null]
["return",null]
["return",{"line":1,"col":58}]
["return",[]]
["return",[["Line 1, characters 26-54","module Test_sexp : sig exception E of int end"]]]
